src/couch/src/couch_bt_engine.erl

```diff
@@ -759,6 +777,45 @@ set_default_security_object(Fd, Header, Compression, Options) ->
     end.
 
 
+% This function is to rollback pf clustered purge.
+% It replaces purge_tree_state and upurge_tree_state with
+% with purged_docs disk pointer that stores only the last purge request
+downgrade_purge_info(Fd, Header) ->
+    {db_header, _, UpSeq, _, IdTreeState, SeqTreeState, LocalTreeState, PSeq,
```

Comment:
Can you reformat this so that it's one field per line and includes names for the _ variables that are ignored? I had to refer back and forth to the various header definitions to understand what was being dropped in each of those wild card matches.

Reply:
Re-formatted usage of db_header no matter whether downgrade_purge_info/2 will be used in pre-Clustered Purge build or not.